pants.toml

```diff
@@ -0,0 +1,43 @@
+# Copyright 2021 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+[GLOBAL]
+pants_version = "2.5.0rc2"
+pantsd = true  # Enable the Pants daemon for better performance.
```

Comment:
This is the default. I wonder if we should stop including this.

Reply:
Probably, I'll remove it. This was a relic from months ago.